src/v2/Apps/Show/ShowApp.tsx

```diff
@@ -52,9 +53,13 @@ export const ShowApp: React.FC<ShowAppProps> = ({ show }) => {
             contextPageOwnerType,
           }}
         >
-          <Box mt={4} mb={[4, 6]}>
-            <ShowInstallShots show={show} mt={4} mb={6} />
-          </Box>
+          <ShowNavigationBanner show={show} mt={2} mb={4} />
```

Comment:
**question:** It looks like we will be showing this navigation banner regardless of how a user arrives at a show page (ie. from `/shows` or the search bar). Is that the intended behavior, or should we only show it if a user arrives at a show page from a fair?

Reply:
I see that in the component it's checking for the fair + href and returning null but I concur that's weird; it feels like the check should be in here rather than the other way around.